website/docs/r/rdb_user_beta.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Could you add this file in https://github.com/scaleway/terraform-provider-scaleway/blob/master/website/scaleway.erb